### ISSUE TITLE
Request design review for Higher-Level Review (Submission Review Screens)

### ISSUE THREAD

**Issue by rileyorr** [department-of-veterans-affairs/va.gov-team#3566]
## Product Name

Higher-level Review Form (Submission Review Screens) 
[Screen 1](https://vsateams.invisionapp.com/share/J8UWM0P3KHM)
[Screen 2](https://vsateams.invisionapp.com/share/B6UWM12NZ4F)
[Screen 3](https://vsateams.invisionapp.com/share/RGUWM1G97N2)

## Checkpoint Stage

[ ] **Design Intent** (for early discussions about needs, priorities, concepts, research planning, and direction)

[X] **Design Review** (for design work validation of best practices and alignment with the va design system)

### Where in the process is this? 

[ ] Initial concept

[X] Initial mockup

[ ] Final mockup

## Point of contact

Riley

## Preferred times

11/21 12:00 pm - 4:00 pm ET
11/22 10:00 am - 11:00 am ET, 12:00 pm - 1:00 pm ET, 3:00 pm - 4:00 pm ET

## Next steps

This issue *automagically* creates a Zenhub ticket (assigned to Emily, Crystabel, and Jennifer (VSA Accessibility Specialist), and VSP will schedule a design review.

**Comment by jenstrickland** [comment]
@CrystabelReiter @emilywaggoner — for visibility towards scheduling a design review. cc @sshein

**Comment by emilywaggoner** [comment]
Got it - thanks @rileyorr! How about Friday at 3pm ET?

**Comment by rileyorr** [comment]
@emilywaggoner That sounds perfect!

**Comment by emilywaggoner** [comment]
Cool. Will send an invite! Are you planning to share the mocks on this ticket ahead of time?

**Comment by rileyorr** [comment]
I definitely can!! I just received some content feedback from Peggy earlier today so I'll incorporate it this afternoon/tomorrow morning and then link the mocks on this ticket!

**Comment by rileyorr** [comment]
I've updated the ticket with the 3 screens that we will be reviewing!

**Comment by emilywaggoner** [comment]
Hey @rileyorr here is a list of the feedback items we discussed on Friday:

- Having multiple address fields is a little confusing to confirm. **Recommendation**: What do you think about adding a header or separating this into its own section, or modifying the form labels? 
- Start date and end date labels seems a little confusing in this context. Screenshot:
<img width="642" alt="Screen Shot 2019-11-25 at 4 11 10 PM" src="https://user-images.githubusercontent.com/1379629/69578532-affc0500-0f9e-11ea-9911-3a64b2004b72.png">

- **Recommendation:** The contested issues section should be a read-only in this context (see the 526 disability application that Peggy referenced during the meeting). Screenshot:
![image](https://user-images.githubusercontent.com/1379629/69578776-2c8ee380-0f9f-11ea-8ac9-34e2a9c182d3.png)
- **Recommendation:** The second "Tinnitus" label seems repetitive, consider removing it once the read only section is in place. Screenshot:
<img width="661" alt="Screen Shot 2019-11-25 at 4 19 56 PM" src="https://user-images.githubusercontent.com/1379629/69578942-855e7c00-0f9f-11ea-815a-7b51bc2a985d.png">
- **Recommendation:** `Representative's availability` and `My availability` having that repeated on multiple lines seems, consider modifying the label to reflect that they are different time slots. Screenshot: 
<img width="629" alt="Screen Shot 2019-11-25 at 4 21 11 PM" src="https://user-images.githubusercontent.com/1379629/69579039-b0e16680-0f9f-11ea-9ce1-c553cc8c70a9.png">

**Comment by jenstrickland** [comment]
Here are a list of items found when auditing the current HLR. I reviewed with @1Copenut to ensure these were worth documenting. Ideally, these issues would be resolved in a redesign of the problematic components during this project, usability tested, and then surfaced to the design system for implementation across the site(s). The timeline of this project may not allow for this, understandably, but any improvements we can make would be helpful, and then backlog items that can't be addressed. 

To experience these issues, turn on VoiceOver on Mac, put your mouse in the h1, then press Control Option A. You’ll hear the page and the options available. (If on Windows, use NVDA in Firefox, which is available to download for free.

=========
When the page is read by a screen reader, there are no cues of interactivity, no indication when one is in an interactive form. There is a risk that a person may get caught inside the editable form elements, not be able to supply info that aligns with the required pattern, and be unable to back out, without completely reloading the page.

Additionally, the progress indicator only says, You are currently on a progress indicator, with no details about number of steps or current step.

<img width="998" alt="Screen Shot 2019-11-27 at 1 43 44 PM" src="https://user-images.githubusercontent.com/57469/69750971-18242580-111c-11ea-9a15-11981c3b4014.png">

=========
This Edit ‘button’ isn’t read as a button.
Screen reader users only hear, “Edit Phone & email” but don’t know they can do anything to enter the edit mode.
<img width="616" alt="Screen Shot 2019-11-27 at 1 46 14 PM" src="https://user-images.githubusercontent.com/57469/69751098-5c172a80-111c-11ea-862a-809de857af59.png">

=========
Focus is on hidden Edit button

What does this mean?
When the screen reader user is in this component, nothing is read out. There’s no indicator telling them where they are, or that the Edit functionality is now available to them.
<img width="627" alt="Screen Shot 2019-11-27 at 1 48 15 PM" src="https://user-images.githubusercontent.com/57469/69751199-954f9a80-111c-11ea-8453-86303d845169.png">

=========
There is a risk that a person may get caught inside the editable form elements, not be able to supply info that aligns with the required pattern, and be unable to back out, without completely reloading the page.
<img width="617" alt="Screen Shot 2019-11-27 at 1 48 55 PM" src="https://user-images.githubusercontent.com/57469/69751235-abf5f180-111c-11ea-8867-617f6b5919f3.png">

=========
There is no guidance that these are interactive, nothing that tells a screen reader user that there is a checkbox to interact with.
When the user is on the button, the AT says, You are currently on a button, but with no connection to what the button is or does.
<img width="618" alt="Screen Shot 2019-11-27 at 1 49 58 PM" src="https://user-images.githubusercontent.com/57469/69751314-d21b9180-111c-11ea-8592-da4b9cd049d7.png">

=========
On the top of this page, the screen reader reads out Yes Yes No No, with no indication of radio input option.
It is possible to select neither radio input and leave the textarea empty and Continue to proceed to the next screen.
<img width="602" alt="Screen Shot 2019-11-27 at 1 51 12 PM" src="https://user-images.githubusercontent.com/57469/69751387-fe371280-111c-11ea-9680-8bd2bd263dd5.png">

=========
On these radio inputs, no indication they are radio buttons is read.
<img width="599" alt="Screen Shot 2019-11-27 at 1 51 56 PM" src="https://user-images.githubusercontent.com/57469/69751474-2cb4ed80-111d-11ea-83f3-fe6d943b61d6.png">

=========
While interacting with the text input, press Enter or Tab, and the screen reader skips the next paragraph of instructions, generates an error that they must choose up to two time periods, but with no guidance on what that means.
<img width="606" alt="Screen Shot 2019-11-27 at 1 53 21 PM" src="https://user-images.githubusercontent.com/57469/69751530-4a825280-111d-11ea-9ca6-ac501d032f47.png">

=========
Error message after form submit

The dialog about the application save is read after the buttons. This may be of more use to a screen reader user if it is read before the buttons.
<img width="618" alt="Screen Shot 2019-11-27 at 1 54 15 PM" src="https://user-images.githubusercontent.com/57469/69751582-6d146b80-111d-11ea-90f8-f1af30e9bbd8.png">

=========
<img width="611" alt="Screen Shot 2019-11-27 at 1 54 46 PM" src="https://user-images.githubusercontent.com/57469/69751609-7ef60e80-111d-11ea-9929-bb5b373033cf.png">

=========
<img width="594" alt="Screen Shot 2019-11-27 at 1 55 16 PM" src="https://user-images.githubusercontent.com/57469/69751647-8f0dee00-111d-11ea-9aa6-e46eea39344a.png">

=========
Focus is on hidden Edit button

There is no message that says the area is interactive.
<img width="723" alt="Screen Shot 2019-11-27 at 1 55 46 PM" src="https://user-images.githubusercontent.com/57469/69751698-a77e0880-111d-11ea-8efa-14ee2c229546.png">

=========
Focus is on now in phone number input but there is no indicator that the input is editable.

What does this mean?
When the screen reader user is in this component, nothing is read out. There’s no indicator telling them where they are, or that the Edit functionality is now available to them.
<img width="616" alt="Screen Shot 2019-11-27 at 1 56 40 PM" src="https://user-images.githubusercontent.com/57469/69751757-c11f5000-111d-11ea-9996-8cfcb6d73abe.png">

=========
It says “You are currently on a progress indicator,” but no read out of what step or number of steps.
<img width="768" alt="Screen Shot 2019-11-27 at 1 57 34 PM" src="https://user-images.githubusercontent.com/57469/69751833-e1e7a580-111d-11ea-97f7-cee442776436.png">

=========

**Comment by YanaRoy** [comment]
@jenstrickland I will be closing this ticket. The comment you added with the purple/yellow notes will be handled separately from this ticket since these are global changes to the forms library. @AZSchneider is in the process of looking into these recommendations and how to move forward.

**Comment by Mottie** [comment]
The above accessibility issues have been moved into new issues:

Global Form issues:

* [SCREENREADER]: Form review card edit button #4701
* [COGNITION]: Form review card edit can not be cancelled #4877
* [SCREENREADER, KEYBOARD]: Selectable issue cards are not accessible #4907
* [KEYBOARD]: Form review edit button focus #4613
* [SCREENREADER]: Form progress bar not reporting steps #4679
* [COGNITION] Save in progress dialog should be above the form navigation #5953

To do:

* HLR specific? (form has changed since this review & I am not able to duplicate the issue)
  > On the top of this page, the screen reader reads out Yes Yes No No, with no indication of radio input option
* HLR Specific? (unable to reproduce)
  > While interacting with the text input, press Enter or Tab, and the screen reader skips the next paragraph of instructions, generates an error that they must choose up to two time periods, but with no guidance on what that means
* Global?
  > On these radio inputs, no indication they are radio buttons is read
  - I did notice that when I move between radio button choices and the form changes (alert messages show/hide) the focus is sometimes lost.
